std/conv.d

```diff
@@ -1208,7 +1209,8 @@ if (!(is(S : T) &&
 */
 private template isSwitchable(E)
 {
-    enum bool isSwitchable = is(typeof({
```

Comment:
and I really don't like this type of change

Reply:
yeah this looks like a bug in D-Scanner